shipment_advice_planner_toursolver/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+In the shipment planner you will see a new option added to the planning methods.
+If you select TourSolver, a TourSolver task per warehouse will be created for
+the pickings you want to plan.
+
+The task will pass through different states by a dedicated cron and at the end
+of the process will create shipment advices according the returned result from
+TourSolver.
```

Comment:
We could add into the roadmap that a webhook could be developed to be notified by toursolver when the result is ready... This could complement the cron. The webhook allowing to get the result as soon as available and the cron ensuring that even if a call to the webhook has been missed, the task process is finalized....